admin_guide/upgrading_citus.rst

```diff
@@ -260,3 +260,8 @@ On Worker Nodes
                                 -d $OLD_PG_DATA -D $NEW_PG_DATA
 
 4. Start the new server.
+5. Add entry to ``pg_dist_local_group``:
```

Comment:
I think the right approach should be to copy/save the metadata values before the upgrade and copy them back after the upgrade (similar to the master node). This fix is a temporary one which works since we aren't yet using pg_dist_local_group. 
@marcocitus What is the interaction with MX? Should we backup all metadata like we do with the coordinator node?